internal/pkg/cli/validate.go

```diff
@@ -120,6 +120,13 @@ func validateEnvironmentName(val interface{}) error {
 	return nil
 }
 
+func validateTaskFamilyName(val interface{}) error {
```

Comment:
That totally makes sense. My concern was that unwrapping it would be inconsistent with what `svc` and `env` is doing (there are `validateSvcName` and `validateEnvironmentName` and they are called at similar places like `validateTaskFamilyName` i.e. at `Validate()` and `o.prompt.Get()`). Any thought?

Reply:
That's a good call. What we currently have for `validateSvcName` is actually a bad behavior. Ideally we should only wrap errors for cross package methods and this error should be something like `task definition %s is not allowed because of %w` as a single layer of error. The same applies to the other similar methods but might need more refactoring work to validate.go.